Sources/Commands/Options.swift

```diff
@@ -83,6 +83,9 @@ struct LocationOptions: ParsableArguments {
     /// Path to the compilation destination describing JSON file.
     @Option(name: .customLong("destination"), help: .hidden, completion: .directory)
     var customCompileDestination: AbsolutePath?
+
+    @Option(name: .customLong("experimental-cc-sdks-path"), help: .hidden, completion: .directory)
```

Comment:
I know it's a hidden option, but could we still unroll the `cc` part somehow or use a different abbreviation? This way it just sounds as if it is related to C compilation.